app/brave_main_delegate.cc

```diff
@@ -190,7 +181,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
                                    kBraveOriginTrialsPublicKey);
   }
 
-  std::string brave_sync_service_url = kBraveSyncServiceURL;
+  std::string brave_sync_service_url = BRAVE_SYNC_ENDPOINT;
 #if defined(OS_ANDROID)
```

Comment:
Maybe we should put here `DCHECK(!brave_sync_service_url.empty())` to prevent accidental having empty url for the persons who will not update the npm config?

Reply:
Or even something like `static_assert(sizeof(BRAVE_SYNC_ENDPOINT) > 0)` to fail it during compile time?

Reply:
we in anyway must edit edit the npm config for api keys , so my comments are not so important